src/org/usfirst/frc/team2706/robot/commands/teleop/GearHandlerClosed.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class GearHandlerClosed extends Command {
```

Comment:
Documentation?? What's the point of this command and when is it used? Also get rid of commented parts if you dont think you're ever going to use them